### PR TITLE
Revert "Update pci-ids and embed it"

### PR DESCRIPTION
**Pull request by rminnich**
This reverts commit 0d741dd84444063f57acd5d41cf053d187960354.

That commit embedded the pci.ids file into the binary.

This seems fine, but I remembered why I did not want to do it that way. The pci.ids format is a bit fragile. A company might make a quick change, not run go test, and end up with a damaged pci.ids embedded in flash. This is very dangerous. With the 'go generate' being required to build with a pci.ids, the parsing will happen as part of build, ensuring that pci.ids is parsed correctly. Waiting until we have burned it into firmware is too risky.

This also restores gen.go, as the bugs we had in 2017 no longer seem to happen.

### CODE REVIEW

**Comment by rminnich** [comment]
interestingly, now it's smaller? 
I wonder if we're including less pci.
```
rminnich@pop-os:~/go/src/github.com/u-root/u-root/cmds/core/pci$ ls -l pci pci.main
-rwxrwxr-x 1 rminnich rminnich 3520786 Jun  5 20:02 pci
-rwxrwxr-x 1 rminnich rminnich 4348846 Jun  5 20:02 pci.main # from main branch
```

**Comment by rminnich** [comment]
more to do here. This just needs more work than I realized. Some of it goes back many years, and some decisions here were made before Go binaries started being so compact.

**Comment by rminnich** [comment]
OK,  a first pass on this, just going to a simple initalized array, gets me here
-rwxrwxr-x 1 rminnich rminnich 2896004 Jun  5 21:46 pci
currently it's 
-rwxrwxr-x 1 rminnich rminnich 4348846 Jun  5 21:46 pci.main

I'll try to wrap this tomorrow.

**Comment by rminnich** [comment]
OK, it did not shrink as much as hoped, but
-rwxrwxr-x 1 rminnich rminnich 3904241 Jun  5 22:21 pci
-rwxrwxr-x 1 rminnich rminnich 4348846 Jun  5 21:46 pci.main

better than no shrinkage, and the runtime pci parsing is gone.

**Review by andrewsun2898 — APPROVED** [review]

File: pkg/pci/gen.go [hunk 1]
```diff
@@ -0,0 +1,100 @@
...
+	"fmt"
+	"io/ioutil"
+	"log"
+	"strconv"
+
+	"github.com/u-root/u-root/pkg/pci"
+)
+
+var (
+	pciidspath = [...]string{"/usr/share/misc/pci.ids"}
```
Comment (10000TB) [inline pkg/pci/gen.go:22]: Is this expected to be present on the machine of build ?

Comment (rminnich) [reply pkg/pci/gen.go:22]: it's a widely distributed, slowly changing but note: there is a supplied pci_ids.go, checked in, which is a common practice with generated files. So if the generate does not work, you still have the pci_ids.go you started with. 

That's how we've done it for about 7 years ... believe it or not! I did not at first! It was my fault.

**Pull request merged**